examples/ssh/main.go

```diff
@@ -14,6 +23,41 @@ import (
 	"github.com/muesli/termenv"
 )
 
+// Available styles.
+type styles struct {
+	bold          lipgloss.Style
+	faint         lipgloss.Style
+	italic        lipgloss.Style
+	underline     lipgloss.Style
+	strikethrough lipgloss.Style
+	red           lipgloss.Style
+	green         lipgloss.Style
+	yellow        lipgloss.Style
+	blue          lipgloss.Style
+	magenta       lipgloss.Style
+	cyan          lipgloss.Style
+	gray          lipgloss.Style
+}
+
+// Create new styles against a given renderer.
+func makeStyles(r *lipgloss.Renderer) styles {
```

Comment:
`r` not used

Reply:
It's being used in the return to create styles no?

Reply:
oh sorry missed that 😄